bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/helpers/RepositoryHelper.java

```diff
@@ -138,4 +144,36 @@ public static <T> IRepository<T> createMemoryComposite(IProvisioningAgent agent,
 		return null;
 	}
 
+	/**
+	 * Returns an unmodifiable list of global shared bundle pools, e.g., as used by
+	 * the Eclipse Installer.
+	 *
+	 * @return an unmodifiable list of global shared bundle pools.
+	 *
+	 * @throws IOException if there are problems loading the pool information.
```

Comment:
On the long term I think we might simply return an empty list in such case?

Reply:
Yes, I wondered too about guarding all exceptions within the method....